server/flags/flags.go

```diff
@@ -49,15 +49,15 @@ const (
 )
 
 // AddTxFlags adds common flags for commands to post tx
-func AddTxFlags(cmd *cobra.Command) *cobra.Command {
-	cmd.PersistentFlags().String(flags.FlagChainID, "testnet", "Specify Chain ID for sending Tx")
-	cmd.PersistentFlags().String(flags.FlagFrom, "", "Name or address of private key with which to sign")
-	cmd.PersistentFlags().String(flags.FlagFees, "", "Fees to pay along with transaction; eg: 10aphoton")
-	cmd.PersistentFlags().String(flags.FlagGasPrices, "", "Gas prices to determine the transaction fee (e.g. 10aphoton)")
-	cmd.PersistentFlags().String(flags.FlagNode, "tcp://localhost:26657", "<host>:<port> to tendermint rpc interface for this chain")
-	cmd.PersistentFlags().Float64(flags.FlagGasAdjustment, flags.DefaultGasAdjustment, "adjustment factor to be multiplied against the estimate returned by the tx simulation; if the gas limit is set manually this flag is ignored ")
-	cmd.PersistentFlags().StringP(flags.FlagBroadcastMode, "b", flags.BroadcastSync, "Transaction broadcasting mode (sync|async|block)")
-	cmd.PersistentFlags().String(flags.FlagKeyringBackend, keyring.BackendOS, "Select keyring's backend")
+func AddTxFlags(cmd *cobra.Command) (*cobra.Command, error) {
+	cmd.Flags().String(flags.FlagChainID, "testnet", "Specify Chain ID for sending Tx")
```

Comment:
why the change to Flags?

Reply:
@fedekunze Ok after reading the cobra docs it makes much more sense :) I added the Persistent flags again and decided to remove the required `chain-id` in the rootCmd, cause otherwise all commands would require a `chain-id`.